app/Http/Controllers/AuthController.php

```diff
@@ -40,6 +40,18 @@ public function loginAction(LoginRequest $request): JsonResponse
         if (is_null($me->password)) {
             throw new FailedLoginException();
         }
+        $invites = ProjectInvite::where('email_address', $me->email_address)->get();
+        foreach ($invites as $invite) {
+            $me->projects()->sync([$invite->project_id => ['is_monitoring' => true]]);
```

Comment:
I believe you should be passing `false` in here as a second parameter to prevent this call from removing the other associated projects (including those added in previous iterations of this loop).